src/lib/pagination/index.ts

```diff
@@ -9,7 +9,7 @@ export const getPaginatedResults = <T>(
     count,
     page: {
       current: page,
-      total: Math.ceil(count / limit),
+      total: Math.max(Math.ceil(count / limit), 1),
```

Comment:
could also be fixed with a ternary like this
```suggestion
      total: count === 0 ? 1 : Math.ceil(count / limit)),
```

Reply:
I do prefer the ternary approach, feels more explicit! But other than that it perfects, good job 🙏

Reply:
updated